test/spec/modules/hmdaFilters.js

```diff
@@ -51,8 +49,8 @@ describe('Filters: hmdaFilters', function() {
         beforeEach(angular.mock.module(function($provide) {
             $provide.value('FileMetadata', mockFileMetadataService);
             $provide.value('HMDAEngine', {
-                getFileSpec: function(year) { return mockFileSpec; },
-                getHmdaJson: function() { return mockHmdaFile }
+                getFileSpec: function() { return mockFileSpec; },
```

Comment:
I wasn't sure this would work for the mock since it doesn't match the signature, but looks like it doesn't matter! Good to know.